src/inference/include/openvino/runtime/intel_gna/gna_properties.hpp

```diff
@@ -0,0 +1,213 @@
+
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief A header for advanced hardware related properties for GNA plugin
+ *        To use in set_property() method of plugins
+ *
+ * @file config.hpp
+ */
+#pragma once
+
+#include "openvino/runtime/properties.hpp"
+
+namespace ov {
+namespace intel_gna {
+
+/**
+ * @brief Metric to get a std::string of GNA Library version, usually in the form
+ * <API_REVISION>.<RELEASE_LINE>.<RELEASE>.<BUILD>
+ */
+static constexpr Property<std::string, PropertyMutability::RO> library_full_version{"GNA_LIBRARY_FULL_VERSION"};
+
+/**
+ * @brief Scale factor that is calculated by user, in order to use static quantisation feature
+ * This option should be used with floating point value serialized to string with decimal separator equals to . (dot)
+ * @details For multiple input case, individual scale factors can be passed, using
```

Comment:
"In the case of multiple inputs, individual scale factors can be provided using the"